Fix typo in embedding_bag_cpu.

kimishpatel opened the pull request:
Summary: Fixed typo.

Differential Revision: D15003373

kimishpatel commented:
@soumith is this safe to land with pytorch-win-ws2016-cuda9-cudnn7-py3 and pytorch_linux_xenial_cuda9_cudnn7_py3_test failing? Not familiar with these so thought will ask.

soumith commented:
i usually go look in the logs. When you click the "Details" button, you can see more.

It looks like the windows one is an infrastructure failure, and the other one is a flaky distributed test (there's a PR being prepared for this).

You can land it

kimishpatel commented:
Thanks @soumith. I did exactly that. Just wanted to make sure it was not something related to my change. Will land it.

kimishpatel committed:
Fix typo in embedding_bag_cpu. (#19432)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/19432

Fixed typo.

Reviewed By: cpuhrsch

Differential Revision: D15003373

fbshipit-source-id: 29e1fbb91cde3e01f183082de2e662f3bf31fcd5
